"app_cfg":
    "loop_rate": 10
    "camera_name": "zed_front"
    "resize_image": False
    "marker_length": 0.05
    "camera_frame_id": "zed_front_left_img"
    "baselink_frame_id": "base_link"
    
    "stream_cfg":
        "resolution": [1920, 1080]
        "frame_rate": 30
    
    "msg_cfg":
        "topics":
            "subscribe":
                "camera_info_topic": "/zed_front/camera_info"
                "rgb_topic": "/zed_front/image/compressed"
                "depth_topic": "/zed_front/depth_image"
                "service_topic": "/initialize_aruco_pose"
                "localization_topic": "/Localization/odometry/filtered_map"
                "tf_static_topic": "/tf_static"

            "publish":
                "rgb_topic": "/aruco_app/zed_front/id_pose/image/compressed"
                "relative_pose_topic": "/aruco_app/zed_front/relative_pose"
                "global_pose_topic": "/aruco_app/zed_front/global_pose"